check_sites.py

```diff
@@ -47,16 +47,20 @@ def find_diff_sets(old_sets, new_sets):
 def main():
     args = sys.argv[1:]
     input_file = 'related_website_sets.JSON'
+    testing_sets = []
     input_prefix = ''
     with_diff = False
-    opts, _ = getopt.getopt(args, "i:", ["data_directory=", "with_diff"])
+    opts, _ = getopt.getopt(args, "i:", ["data_directory=", "with_diff", 
+                                         "testing_sets="])
     for opt, arg in opts:
         if opt == '-i':
             input_file = arg
         if opt == '--data_directory':
             input_prefix = arg
         if opt == '--with_diff':
             with_diff = True
+        if opt == '--testing_sets':
+            testing_sets = arg.split(',')
```

Comment:
```suggestion
            testing_sets.extend(arg.split(','))
```

Just in case people supply the arg multiple times.